public/plugin_nav.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import {
+  observabilityApplicationsID,
+  observabilityIntegrationsID,
+  observabilityMetricsID,
+  observabilityNotebookID,
+} from '../common/constants/shared';
+import { CoreSetup } from '../../../src/core/public';
+import { AppPluginStartDependencies } from './types';
+import { DEFAULT_NAV_GROUPS, DEFAULT_APP_CATEGORIES } from '../../../src/core/public';
+
+export function registerAllPluginNavGroups(core: CoreSetup<AppPluginStartDependencies>) {
+  core.chrome.navGroup.addNavLinksToGroup(DEFAULT_NAV_GROUPS.observability, [
+    {
+      id: observabilityApplicationsID,
+      category: DEFAULT_APP_CATEGORIES.dashboardAndReport,
```

Comment:
OSD core has changed the category to visualizeAndReport but keep the `dashboardAndReport` to keep backward compatible. Could you please change to use visualizeAndReport?